tests/test_scripts/sample_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+wd = "$1"
+file = "$2"
+
+sudo R \
+-e "Sys.setenv(BATCH_ACCOUNT_NAME = '$BATCH_ACCOUNT_NAME')" \
+-e "Sys.setenv(BATCH_ACCOUNT_KEY = '$BATCH_ACCOUNT_KEY')" \
+-e "Sys.setenv(BATCH_ACCOUNT_URL = '$BATCH_ACCOUNT_URL')" \
+-e "Sys.setenv(STORAGE_ACCOUNT_NAME = '$STORAGE_ACCOUNT_NAME')" \
+-e "Sys.setenv(STORAGE_ACCOUNT_KEY = '$STORAGE_ACCOUNT_KEY')" \
+-e "getwd();" \
+-e "setwd('$wd')"
+-e "source('$file')"
```

Comment:
What does this actually test?

Reply:
Will delete. This is supposed to be used for runing our samples